frontend/src/components/NavBar.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './navbar.css'
```

Comment:
this should be `import classes from './NavBar.module.css'` because in React you have to create CSS module files, and within `classes` object, you can then refer to any CSS className vi `classes.{className}`